scale/docs/rest/scheduler.rst

```diff
@@ -23,18 +23,22 @@ There should be one and only one scheduler entry in the database and it is used
 | **Content Type**   | *application/json*                                                                                 |
 +--------------------+----------------------------------------------------------------------------------------------------+
 | **JSON Fields**                                                                                                         |
-+----------------------+-------------------+------------------------------------------------------------------------------+
-| is_paused            | Boolean           | True if the scheduler is paused. This functions like individually pausing    |
-|                      |                   | all nodes but maintains separated state so toggling this back to unpaused    |
-|                      |                   | results in the previous individual node pause state.                         |
-+----------------------+-------------------+------------------------------------------------------------------------------+
-| num_message_handlers | Integer           | The number of message handlers to have scheduled                             |
-+----------------------+-------------------+------------------------------------------------------------------------------+
++--------------------------+---------------+------------------------------------------------------------------------------+
+| is_paused                | Boolean       | True if the scheduler is paused. This functions like individually pausing    |
+|                          |               | all nodes but maintains separated state so toggling this back to unpaused    |
+|                          |               | results in the previous individual node pause state.                         |
++--------------------------+---------------+------------------------------------------------------------------------------+
+| num_message_handlers     | Integer       | The number of message handlers to have scheduled                             |
++--------------------------+---------------+------------------------------------------------------------------------------+
+| system_job_logging_level | String        | The logging level sent to all system tasks that run within                   |
```

Comment:
Rename `system_job_logging_level` to `system_logging_level` and update the description please. This setting is for all Scale system components/pieces, not just jobs.